README.md

```diff
@@ -76,27 +76,27 @@ $ docker run --name zilliz_search_images_demo_web -d --rm -p 8001:80 \
 milvusbootcamp/pic-search-webclient:0.2.0
 ```
 
-In the previous command, WEBSERVER_IP specifies the server IP address that runs pic-search-webserver docker.
+In the previous command line, WEBSERVER_IP specifies the server IP address that runs pic-search-webserver docker.
 
 ### How to perform reverse image search
 
 After deployment, enter ` ${WEBCLIENT_IP}:8001` in the browser to open the interface for reverse image search. **WEBCLIENT_IP** specifies the server IP address that runs pic-search-webclient docker.
 
 <img src="pic/web4.png" width = "650" height = "500" alt="arch" align=center />
 
-Enter the path to the images folder, e.g. /data/images. click Load to load the images. The following screenshot shows the loading process.
+Enter the path to the images folder, e.g. /data/images. Click Load to load the images. The following screenshot shows the loading process.
 
 <img src="pic/web0.png" width = "650" height = "500" alt="arch" align=center  />
 
-> Note: After clicking the load button, the system needs to wait for a while to respond. Please do not click again
+> Note: After clicking the Load button, you need to wait for a while for the system to respond. Please do not make mulitple clicks.
 
-The loading process may take several minutes. The following screenshot shows the interface with images loaded.
+The loading process may take several minutes. The following screenshot shows the interface when the images are successfully loaded.
```

Comment:
It may take several minutes to load the images. The following screenshot shows the interface when image loading is successful.